custom_components/tuya_local/devices/doel_tiplus_litterbox.yaml

```diff
@@ -0,0 +1,249 @@
+  - entity: switch
+    name: Auto Clean
+    category: config
+    icon: "mdi:auto-mode"
+    dps:
+      - id: 105
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Infrared detection
+    category: config
+    icon: "mdi:leak"
+    dps:
+      - id: 111
+        type: boolean
+        name: switch
+  - entity: lock
+    translation_key: child_lock
+    name: Child Lock
+    category: config
+    dps:
+      - id: 114
+        type: boolean
+        name: lock
+  - entity: number
+    name: Clean Wait Time
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 117
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 60
+  - entity: number
+    name: Clean Interval
+    category: config
+    icon: "mdi:update"
+    dps:
+      - id: 118
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 120
+  - entity: binary_sensor
+    name: Smart Cleaning
+    icon: "mdi:refresh-auto"
+    dps:
+      - id: 121
+        type: boolean
+        name: sensor
+  - entity: number
+    name: Capacity Calibration
+    category: config
+    icon: "mdi:trash-can"
+    dps:
+      - id: 123
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 15
+  - entity: number
+    name: Detection Sensitivity
+    category: config
+    icon: "mdi:scale"
+    dps:
+      - id: 124
+        type: integer
+        name: value
+        unit: kg
+        mapping:
+          - scale: 10
+  - entity: switch
+    name: Deep Clean
+    category: config
+    icon: "mdi:shimmer"
+    dps:
+      - id: 125
+        type: boolean
+        name: switch
+  - entity: button
+    name: Level Cycle
+    icon: "mdi:arrow-collapse-down"
+    dps:
+      - id: 126
+        type: boolean
+        name: button
+  - entity: switch
+    name: Soft Mode
+    category: config
+    icon: "mdi:shimmer"
+    dps:
+      - id: 127
+        type: boolean
+        name: switch
+  - entity: switch
+    name: dp_developer_mode
+    category: config
+    icon: "mdi:dev-to"
+    dps:
+      - id: 135
+        type: boolean
+        name: switch
+
+  - entity: select
+    name: Unit Select
+    category: config
+    icon: "mdi:pencil-ruler"
+    dps:
+      - id: 108
+        type: string
+        name: option
+        mapping:
+          - dps_val: "kg"
+            value: "kg"
+          - dps_val: "lb"
+            value: "lb"
+  - entity: sensor
+    name: Status
+    icon: "mdi:toilet"
+    dps:
+      - id: 24
+        type: string
+        name: sensor
+  - entity: select
+    name: Litter Type
+    category: config
+    icon: "mdi:dots-hexagon"
+    dps:
+      - id: 131
+        type: string
+        name: option
+        mapping:
+          - dps_val: "mineral_cat_litter"
+            value: "Mineral"
+          - dps_val: "mixed_cat_litter"
+            value: "Mixed"
+  - entity: binary_sensor
```

Comment:
unknown dps should just be listed as extra attributes on another entity, not entities in their own right.